unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.jslib

```diff
@@ -131,24 +145,26 @@ var WebVideoPlayer = {
     },
 
     WebVideoPlayerTextureUpdate: function (videoId) {
-        const id = Pointer_stringify(videoId);
-
-        if (videos[id].state !== 4) return; //PLAYING
-
-        const textureId = videos[id].textureId;
-
-        GLctx.bindTexture(GLctx.TEXTURE_2D, GL.textures[textureId]);
-
+        const videoData = videos[Pointer_stringify(videoId)];
+        
+        if (videoData.state !== 4) return; //PLAYING
+        
+        if (videoData.useUpdateOptimization && !videoData.newFrame) 
+            return; // No new frame to update
+        else
+            videoData.newFrame = false;
+        
+        GLctx.bindTexture(GLctx.TEXTURE_2D, GL.textures[videoData.textureId]);
         GLctx.texImage2D(
             GLctx.TEXTURE_2D,
             0,
             GLctx.SRGB8_ALPHA8,
```

Comment:
Shouldn't this be `GLctx.RGBA`?
```suggestion
            GLctx.RGBA,
```

Reply:
`RGBA `removes sRGB color space and as a result video looks very overbright. That is why we decided to go first still with `SRGB8_ALPHA8`. In the next sprint we will check if we can use `RGBA `and apply sRGB color space inside Unity via the shader

Reply:
So we aren't getting the performance improvements of RGBA?

Reply:
Sorry, my bad that I didn't update the description of the PR. For now we get improvements from updating texture less time than Unity framerate (only going with video framerate) ➕ using `.m3u8` format on `H.264` codec. 
Combination of these 2 approaches already brings much better performance.

Reply:
but yeah, we will not get performance improvements of RGBA. It needs more research on how to apply sRGB color space. Here you see to images of Earth - first one with sRGB color space applied, and second without (RGBA used for internalFormat). You can see how much more overbright is on the second image
![image](https://user-images.githubusercontent.com/35366872/199936178-ab808ff4-0dc3-4a48-ab21-918752964627.png)
![image](https://user-images.githubusercontent.com/35366872/199936207-bf835a9d-b3e0-4528-a242-f329e1b6dbde.png)

Reply:
I'd prioritize performance over all things. We can always do what we always do: setting emissive texture to make it bright.

@aixaCode

Reply:
Or forcing gamma mode to the texture!

Reply:
https://docs.unity3d.com/Manual/LinearRendering-LinearTextures.html#DisablingsRGBSampling

Reply:
this how it looks without sRGB applied 
![image](https://user-images.githubusercontent.com/35366872/200297080-6e02b99c-b0e2-4190-8aa2-409c7f03e16e.png)

Reply:
I would suggest that we do a performance check RGBA vs sRGB. If there is degradation then I would revert this change and keep it as it was for the last festival and include just other performance improvements. 
We can fix the washed colors as @menduz suggested, or with applying the correction in the shader as the next interaction if we have time.

Reply:
in Chrome sRGB brings additional 12 ms (minimum) to the GPU, while with RGBA I get max 1 ms on the GPU. 
![image](https://user-images.githubusercontent.com/35366872/200308523-de889879-09bf-45cd-bdb6-8cac4d38413b.png)

Reply:
however, even with this GPU overhead, comparing current branch improvement (sRGB) with the dev, it removes full Frame Drops for me (only Partially Droped frames remained)
![image](https://user-images.githubusercontent.com/35366872/200309778-bfd69e06-b97f-4b85-9f41-6d6030aa5f1b.png)

Reply:
I'd say lets go with the perf improvement and ask the team to use emissive textres 

This was last year's performance, we are far from it https://twitter.com/potradamus/status/1588768424075096065?s=46&t=xfSvwNO1Jhx9otIsDhmF0w